packages/typespec-azure-resource-manager/lib/arm.foundations.tsp

```diff
@@ -308,20 +308,20 @@ model ArmTagsProperty {
 /**
  * The kind of entity that created the resource.
  */
-// NOTE: This is how the enum is named in types.json
+// NOTE: This is how the extensible enum is named in types.json
 @doc("The kind of entity that created the resource.")
-enum createdByType {
+union createdByType {
```

Comment:
Those should have `string` as a variant

Reply:
one issue with doing this right now is that if someone was spreading those in their enum it will fail vs the other way around works fine.